packages/openapi-generator/src/parser/operation.ts

```diff
@@ -15,15 +14,14 @@ export function parseOperation(
   // TODO: What does the OpenApi generator do in this case?
   const requestBody = parseRequestBody(operation.requestBody, refs);
   const parameters = parseParameters(operation, refs);
-  const operationName = parseOperationName(operation, pattern, method);
 
   return {
     ...operation,
     pattern,
     method,
     requestBody,
     parameters,
-    operationName
+    operationName: operation.operationId!
```

Comment:
how about rename it to `operationId`?

Reply:
Actually you are totally right, and with this I could even make the type a bit nicer.